src/ZstdSharp/ZstdSharp.csproj

```diff
@@ -44,7 +44,6 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="InlineIL.Fody" Version="1.8.0" PrivateAssets="all" />
     <PackageReference Include="InlineMethod.Fody" Version="0.7.1" PrivateAssets="all" />
```

Comment:
I would like to drop `InlineMethod.Fody` too; it can bloat generated IL and I'm pretty sure that modern frameworks can optimize the code well without it. What do you think?